### PR TITLE
[DNM]: fix typo and make logic related code more compact 

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update key monitor metrics in both TiCDC document and official document

### CODE REVIEW

File: cdc/http_status.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 	"go.uber.org/zap"
 )
 
-func (s *Server) startStatusHTTP() error {
+func (s *Server) createStatusHTTP() error {
```
Comment (overvenus) [inline cdc/http_status.go:41]: Could you define a StatusServer struct to decoupling createStatusHTTP from Server?

```go
statusServer := createStatusServer()
s.statusServer = statusServer
```

**Review by overvenus — COMMENTED** [review]
rest lgtm

**Comment by JinLingChristopher** [comment]
```
[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation

[2021-06-01T12:30:13.105Z] /tmp/go-build4275524149/b947/owner.test: Symbol `stderr' causes overflow in R_X86_64_PC32 relocation
```
This seems like new kind of error, maybe cause by new version of go ?

**Comment by JinLingChristopher** [comment]
@leoppro  please take a look at the unit-test issue, it seems like caused by sink.

**Comment by hi-rustin** [comment]
@3AceShowHand Can you resolve the conflict? So we can move forward with the PR! Thank you!

**Pull request closed without merging**